interpreter/js/src/main/scala/sigmastate/crypto/Platform.scala

```diff
@@ -28,6 +29,14 @@ object Platform {
     jsShorts.toArray[Short].map(x => x.toByte)
   }
 
+  def bytesToJsShorts(bytes: Array[Byte]): js.Array[Short] = {
```

Comment:
ScalaDoc missed here and for all other new methods in the class

Reply:
fixed